server.js

```diff
@@ -2,19 +2,26 @@ import express from 'express';
 import cors from 'cors';
 import mongoose from 'mongoose';
 import 'dotenv/config';
-import router from './api/index.js';
+import './config/config-passport.js';
+import usersRouter from './routes/users.routes.js';
+import contactsRouter from './routes/contacts.routes.js';
 
 const app = express();
 
 app.use(express.json());
 app.use(cors());
-app.use('/api', router);
+app.use('/api', usersRouter);
+app.use('/api', contactsRouter);
 
 app.use((_, res, __) => {
   res.status(404).json({
     status: 'error',
     code: 404,
-    message: 'Use api on routes: /api/contacts',
+    message: `Use api on routes: 
+    /api/signup - registration user {email, password}
```

Comment:
1. Raczej nie robimy dokumentacji w miejscu obsługi błędu.
2. Te endpointy są nieprawidłowe bo masz np /api/users/signup albo /api/users/current.

Ja bym tu zostawił poprosutu coś w stylu Not found / The given endpoint does not exist

Reply:
W tym stylu była zrobiona przykładowa appka w konspekcie GoIT. Poprawiłem.